admin/Dockerfile

```diff
@@ -1,23 +1,52 @@
-# debian:bookworm 2024-01-12
-FROM debian@sha256:b16cef8cbcb20935c0f052e37fc3d38dc92bfec0bcfb894c328547f81e932d67
+FROM debian:bookworm
```

Comment:
IIRC this should stay at a pinned version for reproducibility purposes (though setting it just to bookworm would save maintenance overhead).

Reply:
(Looks like we stopped pinning Focal as well around the time of @legoktm redoing the build process - we could probably chat about that and either consistently pin or consistently not.)

Reply:
(@legoktm clarified their reasoning offline)